From: Dan Carpenter <dan.carpenter@oracle.com>
Date: Wed, 8 Nov 2017 11:38:20 +0300
Subject: [PATCH] scsi: mpt3sas: remove a stray KERN_INFO
References: bsc#1067965
Git-commit: 268eb4989410619c39b041d453a05a71b26b24c6
Patch-Mainline: v4.15-rc1

pr_info() has a KERN_INFO already so the second KERN_INFO isn't needed.

Signed-off-by: Dan Carpenter <dan.carpenter@oracle.com>
Acked-by: Sathya Prakash Veerichetty <sathya.prakash@broadcom.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 drivers/scsi/mpt3sas/mpt3sas_scsih.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/scsi/mpt3sas/mpt3sas_scsih.c b/drivers/scsi/mpt3sas/mpt3sas_scsih.c
index 18d1804..362f406 100644
--- a/drivers/scsi/mpt3sas/mpt3sas_scsih.c
+++ b/drivers/scsi/mpt3sas/mpt3sas_scsih.c
@@ -8684,7 +8684,7 @@ static int _scsih_set_satl_pending(struct scsi_cmnd *scmd, bool pending)
 
 			if (pcie_device->handle == pcie_device_pg0->DevHandle)
 				goto out;
-			pr_info(KERN_INFO "\thandle changed from(0x%04x)!!!\n",
+			pr_info("\thandle changed from(0x%04x)!!!\n",
 			    pcie_device->handle);
 			pcie_device->handle = pcie_device_pg0->DevHandle;
 			if (sas_target_priv_data)
-- 
1.8.5.6

